lib/maintainers.nix

```diff
@@ -95,4 +95,10 @@
     githubId = 30784060;
     name = "Pedro Sánchez";
   };
+  refaelsh = {
+    email = "refaelsh@pm.me";
+    github = "refaelsh";
+    githubId = 2750775;
+    name = "Refael Sheinker";
+  };
```

Comment:
Good idea. Should have thought about it myself.
In your opinion, what would be a better way to achieve this:
1. Reverse changes to `mainteiner.nix` on `add-nvim-org-mode`. Then update `mainteiner.nix` on `add-nvim-org-mode` and make a new commit on `add-nvim-org-mode` with a commit message `lib/maintainers: add rafaelsh`.
2. Open a new branch named `add refresh as maintainer`, update `mainteiner.nix`, open a new PR for the `add refresh as maintainer` branch?

Reply:
Depends on you're git tool of choice. I usually use lazygit and select the code from the git diff I want to break out into a new commit with some keybinds. Otherwise, you can do an interactive rebase  or even simpler just do a git mixed reset to previous commit so all your changes are in the local working state but uncommitted and then commit the changes separately.